README.md

```diff
@@ -175,6 +170,20 @@ rescue WebAuthn::VerificationError => e
 end
 ```
 
+## Attestation Statement Formats
+
+| Attestation Statement Format | Supported? |
+| -------- | :--------: |
+| packed (self attestation) | Yes |
+| packed (x5c attestation) | Yes |
+| packed (ECDAA attestation) | No |
+| tpm (x5c attestation) | No |
+| tpm (ECDAA attestation) | No |
+| android-key | Yes |
+| android-safetynet | Yes |
+| fido-u2f | Yes |
+| none | Yes |
```

Comment:
I think it'd be fair to mention here we don't support the FIDO metadata service yet, which is needed for most of these formats to verify the returned trust paths is trustworthy (per [7.1. Registering a New Credential](https://www.w3.org/TR/webauthn/#registering-a-new-credential) steps 15 and 16).

Reply:
You're right, this is a good place to mention that.
Thanks.